secure/key-manager/src/lib.rs

```diff
@@ -67,7 +67,7 @@ pub enum Error {
     #[error("Key mismatch, config: {0}, storage: {0}")]
     ConfigStorageKeyMismatch(Ed25519PublicKey, Ed25519PublicKey),
     #[error("Data does not exist: {0}")]
-    DataDoesNotExist(&'static str),
+    DataDoesNotExist(String),
```

Comment:
A useful trick if you want to avoid an allocation on some paths while using a static str in others would be to use a `Cow<'static, str>`

Reply:
Thanks @bmwill! I think a String is probably fine for now (and is consistent with the rest of our errors), but good to know for the future!